src/mpid/ch4/include/mpidpre.h

```diff
@@ -25,6 +25,8 @@
 #include "mpl_uthash.h"
 #include "ch4_coll_params.h"
 
+extern MPIDI_CH4_coll_algo_container_t mpir_fallback_container;
```

Comment:
@raffenet Ken, for some cases we need dummy container to fallback to MPIR logics. In the previous version we dynamically allocated memory for this, but it was unclear where to release this memory since this container should be passed to underlying levels. This led to messages like "## [0] 24 at [0x0000000001586490], ./src/mpid/ch4/src/ch4_coll_select.h[293]" during tests execution (I believe that is due to debug build) and consequently test was counted as failed. That is why we decide slightly change logic here. As for the name of the type we'll correct it.